packages/forms-web-app/src/dynamic-forms/dynamic-components/date/question.js

```diff
@@ -81,7 +84,7 @@ class DateQuestion extends Question {
 			const answerDateString = journey.response.answers[this.fieldName];
 
 			if (answerDateString) {
-				const answerDate = new Date(answerDateString);
+				const answerDate = convertUTCToUK(answerDateString);
 				day = `${answerDate.getDate()}`.slice(-2);
 				month = `${answerDate.getMonth() + 1}`.slice(-2);
 				year = `${answerDate.getFullYear()}`;
```

Comment:
Is this tested? the `get*` methods are local time (which for the server is UTC) - you might need to use `formatInTimeZone` for each element?

Reply:
Have added some tests, is that enough to prove it works, or have I misunderstood how convertUTCToUK works?